### ISSUE TITLE
Turn on Logins for all users (not just Nightly)

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/fenix#7796]
### Why/User Benefit/User Problem
Currently, the Logins feature is Nightly-only, but we should turn it on for all users.

### What/Requirements
- [x] ~~#6704 Enable FxA state encryption for all users~~ Will postpone this, and move forward without FxA encryption state - see #8324
- [x] #7957 Enable Login management preference (View Logins) by removing Nightly flag
- [ ] When GV 74 becomes beta (2/10) and AC picks it up, enable Login Save and Autofill.

### Acceptance Criteria (how do I know when I’m done?)
Dependencies are fulfilled, and logins are enabled for release users.

**Comment by liuche** [comment]
This has no more dependencies it's waiting on.

**Comment by ekager** [comment]
We want the fix(es) for #8324 to land first so we don't have release users start using the encrypted storage and then be switched to plaintext.

**Comment by ekager** [comment]
GV74 Beta builds should be available soon, then AC will have a merge day where the Gecko Engine Nightly files will be turned into Gecko Engine Beta, so this should require no additional work from us in AC to make sure logins are working once the GV Beta Version is bumped.

**Comment by ekager** [comment]
Merge day PR: https://github.com/mozilla-mobile/android-components/pull/5912

**Comment by sv-ohorvath** [comment]
Verified that saved logins management and autofill are enabled on Beta 4.0.
